### PR TITLE
FFWEB-1884 update code listings

### PR DESCRIPTION
**Pull request by omikron-fp**
Also contains styling for `blockquote` to use it for highlighted boxes such as "Notes", "Important", etc.
Additionally fixes some typos and formatting.

### CODE REVIEW

File: markdown/4.x/en/faq.md [hunk 1]
```diff
@@ -8,7 +8,7 @@ Latest.
 ---
 
 #### Question:
-Can I contribute to the future development of WebComponents?
+Can I contribute to the future development of Web Components?
```
Comment (KrzysztofKozlowskiOBSS) [inline markdown/4.x/en/faq.md:11]: `Web Components` -> `FACT-Finder Web Components`

**Review by KrzysztofKozlowskiOBSS — APPROVED** [review]
One not directly related concern is whether we should maintain the article explaining how to enrich FF response by proxying FF requests. IIRC this method was deprecated as some performance degradations appeared.

**Comment by omikron-fp** [comment]
Rebased. See latest fixups for changes.
Removal of the proxy page shall be considered separately.

**Comment by KrzysztofKozlowskiOBSS** [comment]
> Rebased. See latest fixups for changes.
> Removal of the proxy page shall be considered separately.


👌

**Review by KrzysztofKozlowskiOBSS — APPROVED** [review]

**Pull request merged**